### ISSUE TITLE
JustEnoughResources just stops working with this mod installed.

### ISSUE THREAD

**Issue by adalada** [Abastro/StellarSky#115]
I wouldn't know why but I have 40 mods and when I add just this one (and the API), JustEnoughResources stops working.

**Comment by WarriorsMustang17** [comment]
Yep, same issue here. Installed this and nothing works with jer. Weird because it doesn't have anything to do with it.
